docs/package.json

```diff
@@ -3,14 +3,13 @@
   "private": true,
```

Comment:
q: will this be automatically released when merged to `main`? I'm just wondering if there is any risk of breaking the published docs after we land this

Reply:
Shouldn’t break anything unless I’ve missed something major. The files were just being copied across during the build process, so removing that stops any content being overwritten. And I’ve amended both versions of the files to be better suited to their destinations… that should be enough?

Reply:
Sounds OK - I'm not too familiar with the existing process though, just wanted to double-check